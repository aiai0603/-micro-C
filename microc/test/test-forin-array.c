Void main(Int n) {
  
    Int a[10];
    a[3] =1;
    a[4] =2;
    a[5] =3;
    a[6] =4;
    a[7] =1;
    Int i;
    
    for i in (a[3],a[7])
    {
        print("%d",i);
    }




}

