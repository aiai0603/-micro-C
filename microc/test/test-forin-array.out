24 19 1 5 25 15 10 14 0 9 2 13 0 11 1 11 0 3 1 0 1 12 15 -1 13 0 11 1 11 0 4 1 0 2 12 15 -1 13 0 11 1 11 0 5 1 0 3 12 15 -1 13 0 11 1 11 0 6 1 0 4 12 15 -1 13 0 11 1 11 0 7 1 0 1 12 15 1 13 0 12 1 13 0 11 1 11 0 3 1 11 12 15 -1 16 140 13 0 12 1 13 0 11 1 11 0 3 1 11 13 0 13 1 11 1 11 12 15 -1 13 0 13 1 13 0 13 1 11 0 1 1 12 15 -1 13 0 12 1 11 22 15 -1 13 0 13 1 11 0 7 6 17 94 13 0 13 1 0 3 12 21 14 21 13